dodiscover/replearning/gin.py

```diff
@@ -0,0 +1,113 @@
+"""
+Wrapper for the GIN algorithm in causal-learn
+TODO: Need type hints
+"""
+from causallearn.search.HiddenCausal.GIN.GIN import GIN as GIN_
+from pywhy_graphs import CPDAG
+
+class GIN:
+    """Wrapper for GIN in the causal-learn package.
+
+    Parameters
+    ----------
+    indep_test_method : str
```

Comment:
I would rename this perhaps to `ci_estimator_method`, or  `ci_estimator_clearn` to keep in line with the naming followed in constraint-based algos?

Reply:
[14bfaa0](https://github.com/py-why/dodiscover/pull/72/commits/14bfaa001cfd9233543cb782094c4a0f4e572981)